components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/GranularityModeSelector.jsx

```diff
@@ -158,57 +95,86 @@ export default class GranularityModeSelector extends React.Component {
             case 'From Year':
                 granularityOptions = ['1 Year'];
                 break;
+            default:
+                // do nothing
         }
         return granularityOptions;
     }
 
     getCustomSelectButton() {
-        let selectedTimeRange = '';
-        if (location.search.length > 1) {
-            selectedTimeRange = JSON.parse(decodeURI(location.search.substr(1))).tr;
-        }
-        let customButton = selectedTimeRange === 'custom' ?
-            <FlatButton
-                onClick={this.handleClick}
-                label='Custom'
-                style={{backgroundColor: '#383838'}}/> :
-            <FlatButton
-                onClick={this.handleClick}
-                label='Custom'/>;
+        const {
+            getDateTimeRangeInfo, options, onChangeCustom, theme,
+        } = this.props;
+        const { open, anchorEl } = this.state;
+        const selectedTimeRange = getDateTimeRangeInfo().tr || '';
+        const customButton = selectedTimeRange === 'custom'
+            ? (
```

Comment:
wrong formatting

Reply:
This file is removed with PR https://github.com/wso2/analytics-solutions/pull/270